manifest.ts

```diff
@@ -0,0 +1,29 @@
+import {AppsBindings} from 'mattermost-redux/constants/apps';
+
+type RequestedPermissions = 'act_as_user' | 'act_as_bot'
+type RequestedLocations = AppsBindings.POST_MENU_ITEM | AppsBindings.CHANNEL_HEADER_ICON | AppsBindings.COMMAND | AppsBindings.IN_POST
+
+export type Manifest = {
```

Comment:
This manifest type should exist in mm–redux

Reply:
https://github.com/mattermost/mattermost-app-zendesk/issues/11#issuecomment-751797277